carbon-location.html

```diff
@@ -148,8 +148,8 @@
         }
       },
 
-      __computeRoutePath: function(path, hash, useHashAsPath) {
-        return useHashAsPath ? hash : path;
+      __computeRoutePath: function() {
+        return this.useHashAsPath ? this.__hash : this.__path;
```

Comment:
Good question! It appeared like it might while I was deep in the process of debugging things, but all tests pass just fine with these observer args restored. It should be a bit faster this way, and it's fewer lines to boot, so I've undone these changes.